### DISCUSSION TITLE
Laravel Volt compatibility (with Folio and single-file component)

### DISCUSSION THREAD

**Discussion by cmizzi** [livewire/livewire#6263] [7. Bugs (archived)]
### Livewire version

v3.0.0-beta.7

### Which PHP version are you using?

PHP 8.2

### Steps To Reproduce

Create a fresh project, install the latest Laravel Octane, Volt, and Folio dependencies. Next, create a Folio page containing the following content in `resources/views/pages/users/index.blade.php`: 

```php
<?php

\Livewire\Volt\state(count: 0);

$increment = fn() => $this->count++;

?>

<html>
<head>
    <title>Todos</title>
</head>

<body>
    @volt
    <div>
        <h1>{{ $count }}</h1>
        <button wire:click="increment">+</button>
    </div>
    @endvolt
</body>
</html>
```

Then, start the Octane server, and go to `localhost:8000/users`.

### Additional information

Following [laravel/octane#729](https://github.com/laravel/octane/issues/729).

### Octane Version

2.0.4

### Laravel Version

10.17.1

### PHP Version

8.1

### What server type are you using?

Swoole

### Server Version

5.0.3

### Description

On a fresh project, I'm unable to make Laravel Octane and Laravel Volt (+ Laravel Folio) work together. On the first request, Livewire's asset is correctly injected into the template, but on subsequent requests, the asset disappears and the component doesn't work.

![image](https://github.com/laravel/octane/assets/340143/d2c60da6-dbc8-4b9c-8647-0a5615974fdf)
![image](https://github.com/laravel/octane/assets/340143/3e5900f7-a6e7-4c54-9712-341a683e4e5d)

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by joshhanley** [comment]
@cmizzi thanks for reporting!

Any chance you could test it with just Octane and Livewire V3 (no Volt or Folio)?

As we need to know if it's an underlying Livewire issue or whether it might be a Volt or Folio problem (which is handled by the Laravel team, not us, on their dedicated repos).

Thanks!

**Reply by cmizzi** [reply]
I just tested with `octane` and `livewire`. The issue still occurs without Volt and Folio. I simply created a brand new Laravel project, installed `octane` and `livewire` (beta 3), and followed [the quickstart guide](https://livewire.laravel.com/docs/quickstart). Once Octane has started, the first request works (Livewire's asset is correctly injected). But after refreshing the page, the asset disappears.

I produced [an example project](https://github.com/cmizzi/livewire-6263). You can test it using Laravel Sail, everything is already configured to reproduce the issue.

```bash
composer i

sail up -d
# or 
php -d variables_order=EGPCS /var/www/html/artisan octane:start --server=swoole --host=0.0.0.0 --port=80 --watch
```

Then go to [localhost/counter](http://localhost/counter). The first request will work, the second will fail to inject Livewire's asset.

**Reply by joshhanley** [reply]
@cmizzi thanks for confiriming! There has been some updates to octane support in #6268 so hopefully that fixes your issue. Can you test it when the next release is tagged (should be today) and let us know? Thanks!

**Reply by cmizzi** [reply]
It seems to work pretty well! Thank you for your quick fix 👍
